core/src/main/java/io/grpc/internal/MessageDeframer.java

```diff
@@ -153,11 +155,12 @@ public void request(int numMessages) {
   }
 
   /**
-   * Adds the given data to this deframer and attempts delivery to the sink.
+   * Adds the given data to this deframer and attempts delivery to the listener.
    *
    * @param data the raw data read from the remote endpoint. Must be non-null.
    * @param endOfStream if {@code true}, indicates that {@code data} is the end of the stream from
-   *        the remote endpoint.
+   *        the remote endpoint.  End of stream should not be used in the event of an error, such as
```

Comment:
s/error/transport error/ ? Or really any other clarification that makes it obvious we aren't talking about the grpc status code.